feathr_project/setup.py

```diff
@@ -83,6 +83,8 @@
         "avro<=1.11.1",
         "azure-storage-file-datalake<=12.5.0",
         "azure-synapse-spark<=0.7.0",
+        # Synapse's aiohttp package is old and does not work with Feathr. We pin to a newer version here.
+        "aiohttp==3.8.3",
```

Comment:
Suggest to clean up aiohttp notes in [product_recommendation_demo_advanced.ipynb](https://github.com/feathr-ai/feathr/blob/66480d7d9edbc0b8e4d7307e0fe6e5bad66a11f5/docs/samples/product_recommendation_demo_advanced.ipynb#L215) as user does not need to care about this any more.

Reply:
it's already done in my another PR refactoring the notebooks! Thank you for the comment.